toxcore/TCP_connection_test.cc

```diff
@@ -0,0 +1,16 @@
+#include "tcp_connection.h"
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+// TODO(Jfreegman) make this useful or remove it after NGC is merged
+TEST(TCP_connection, NullTest)
+{
+    (void)tcp_connections_count(nullptr);
```

Comment:
This could be used in `tcp_connected_relays_count`. No need to use it everywhere, but at least in one function.